conf.d/python.d.conf

```diff
@@ -0,0 +1,4 @@
+#plugins_config_dir=
+#plugins_dir=
+debug=no
+example=no
```

Comment:
Is there a way to add comments?
If yes, it would be nice if you could add a short description.

This file will also be overwritten by `make install`. I have added a work-around in the installer for `charts.d.conf`, probably we could also add this file.